third_party/transducer/CMakeLists.txt

```
@@ -30,9 +17,11 @@ ELSE()
 ENDIF()
 
 ADD_LIBRARY(warprnnt STATIC submodule/src/rnnt_entrypoint.cpp)
+target_include_directories(warprnnt PUBLIC submodule/include)
+set_target_properties(warprnnt PROPERTIES PUBLIC_HEADER submodule/include/rnnt.h)
```

nit: the commands are capitablized in this file :)
```suggestion
TARGET_INCLUDE_DIRECTORIES(warprnnt PUBLIC submodule/include)
SET_TARGET_PROPERTIES(warprnnt PROPERTIES PUBLIC_HEADER submodule/include/rnnt.h)
```

---

This is intended. only ancient CMake should use all capitals. I could have lowercased others but I left them untouched.

https://stackoverflow.com/a/32783814